src/EnergyPlus/SingleDuct.cc

```diff
@@ -5347,57 +5347,40 @@ void CalcATMixer(EnergyPlusData &state, int const SysNum)
     // PURPOSE OF THIS SUBROUTINE
     // Calculate the mixed air flow and conditions in the air terminal mixer
 
```

Comment:
Update CalcATMixer according to changes in #11004. This change causes diffs and this branch was created to isolate those diffs associated with the refactor of this function.